Swiftification.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Swiftification"
-  s.version      = "8.0.0"
+  s.version      = "9.0.0"
```

Comment:
This really isn't a huge deal, but do we need a full version bump for a minor change?